spec/operators/bufferToggle-spec.ts

```diff
@@ -1,352 +1,414 @@
+      const e1 = hot('--a--^---b---c---d---e---f---g---h------|    ');
+      const e2 = cold('    --x-----------y--------z---|            ');
+      const subs = '       ^-------------!                         ';
+      const closings = [
+        cold('             ---------------s--|                     '),
+        cold('                         ----(s|)                    '),
+        cold('                                  ---------------(s|)')
+      ];
+      const closeSubs0 =  '--^-----------!                         ';
+      const expected =    '--------------#                         ';
+
+      let i = 0;
+      const result = e1.pipe(
+        bufferToggle(e2, () => {
+          if (i === 1) {
+            throw 'error';
+          }
+          return closings[i++];
+        })
+      );
 
-    expectObservable(result).toBe(expected);
-    expectSubscriptions(e1.subscriptions).toBe(subs);
-    expectSubscriptions(closings[0].subscriptions).toBe(closeSubs0);
-    expectSubscriptions(closings[1].subscriptions).toBe([]);
-    expectSubscriptions(closings[2].subscriptions).toBe([]);
+      expectObservable(result).toBe(expected);
+      expectSubscriptions(e1.subscriptions).toBe(subs);
+      expectSubscriptions(closings[0].subscriptions).toBe(closeSubs0);
+      expectSubscriptions(closings[1].subscriptions).toBe([]);
+      expectSubscriptions(closings[2].subscriptions).toBe([]);
+    });
   });
 
   it('should propagate error emitted from a closing', () => {
-    const e1 = hot('--a--^---b---c---d---e---f---g---h------|');
-    const e2 =     cold('--x-----------y--------z---|        ');
-    const subs =        '^             !                     ';
-    const closings = [
-      cold(             '---------------s--|               '),
-      cold(                         '#                     ')];
-    const closeSubs =  ['  ^           !                     ',
-                      '              (^!)                  '];
-    const expected =    '--------------#                     ';
-
-    let i = 0;
-    const result = e1.pipe(bufferToggle(e2, () => closings[i++]));
-
-    expectObservable(result).toBe(expected);
-    expectSubscriptions(e1.subscriptions).toBe(subs);
-    expectSubscriptions(closings[0].subscriptions).toBe(closeSubs[0]);
-    expectSubscriptions(closings[1].subscriptions).toBe(closeSubs[1]);
+    testScheduler.run(({ hot, cold, expectObservable, expectSubscriptions }) => {
+      const e1 = hot('--a--^---b---c---d---e---f---g---h------|');
+      const e2 = cold('    --x-----------y--------z---|        ');
+      const subs = '       ^-------------!                     ';
+      const closings = [
+        cold('             ---------------s--|                 '),
+        cold('                         #                       ')
+      ];
+      const closeSubs =  [
+        '                  --^-----------!                     ',
+        '                --------------(^!)                    '
```

Comment:
This doesn't seem to be aligned. I think the left should be aligned with the `^` in `e1`.